src/main/java/kr/codesqaud/cafe/controller/UserController.java

```diff
@@ -0,0 +1,58 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.dto.UserDTO;
+import kr.codesqaud.cafe.repository.MemoryUserRepository;
+import kr.codesqaud.cafe.repository.UserRepository;
+import kr.codesqaud.cafe.service.UserService;
+import kr.codesqaud.cafe.user.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
```

Comment:
```suggestion
import java.util.List;
```
`import`는 떨어지지 않도록 작성해주세요.